js/views/pageNavVw.js

```diff
@@ -490,14 +567,20 @@ module.exports = baseVw.extend({
     var $target = $(e.target),
         $popMenu,
         closeHandler;
-
-    if (!(
-        $target.hasClass('popMenu') ||
+        
+    if (!$target.hasClass('js-navAddressBar') && this.suggestionsVisible) {
+      app.hideOverlay();
+      this.hideSuggestions();
+    } else if ($target.hasClass('js-navAddressBar') || !($target.hasClass('popMenu') ||
```

Comment:
This logic shouldn't be changed, it was 

if (!(
        $target.hasClass('popMenu') ||
        $target.parents('.popMenu').length ||
        $target.is('[data-popmenu]') ||
        $target.parents('[data-popmenu]').length
      )) {

Now it's:

if false $target.hasClass('popMenu' or if true $target.parents('.popMenu').length ||
        $target.is('[data-popmenu]') ||
        $target.parents('[data-popmenu]').length

That reverses what should happen.